modules/ROOT/pages/dataweave-cookbook-perform-basic-transformation.adoc

```diff
@@ -5,11 +5,11 @@ endif::[]
 :keywords: studio, anypoint, transform, transformer, format, xml, json, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure
 // :page-aliases: 4.3@mule-runtime::dataweave-cookbook-perform-basic-transformation.adoc
 
-This simple DataWeave example changes the XML input to JSON output by using selector expressions without any functions. It also changes the order and names of some of the fields. Note that more complex transformations usually require the use of the `map` or `mapObject` function.
+These simple DataWeave example changes the XML input to JSON output. Note that more complex transformations usually require the use of the `map` or `mapObject` function.
```

Comment:
shouldn't it be `this` instead of `these` here?

Reply:
There are two examples in the page. I'll change _example_ to _examples_.

Reply:
👍